client/src/App.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react';
```

Comment:
right now, 
in package.json it starts index.js.
index.js then calls App.js which loads index "component"
from index.js directly load index.jsx (the component)
Also change the name if index.jsx to something else like home